pypesto/optimize/optimizer.py

```diff
@@ -662,6 +666,61 @@ def is_least_squares(self):
         return False
 
 
+class ScipyDifferentialEvolutionOptimizer(Optimizer):
+    """
+    Global optimization using scipy's differential evolution optimizer.
+    Package homepage: https://docs.scipy.org/doc/scipy/reference/generated\
+        /scipy.optimize.differential_evolution.html
+
+    Parameters
+    ----------
+    options:
+        Optimizer options that are directly passed on to scipys' optimizer
+
+
+    Examples
+    --------
+    Arguments that can be passed to options:
+
+    maxiter:
+        used to calculate the maximal number of funcion evaluations by
+        maxfevals = (maxiter + 1) * popsize * len(x)
+    popsize:
+        population size, default value 15
+    """
+
+    def __init__(self, options: Dict = None):
+        super().__init__()
+
+        if options is None:
+            options = {'maxiter': 66}
```

Comment:
```suggestion
            options = {'maxiter': 100}
```
I agree that the scipy default of 1000 (--> neval = 1000 * 15 * len(x)) is too long. Maybe this is a good default. 66 is a bit arbitrary.

Reply:
Done